scripts/latest.txt

```diff
@@ -1,10 +1,10 @@
 {
     "version": "2.15",
-    "url":     "https://docs.wagtail.io/en/stable/releases/2.15.html",
```

Comment:
Question: This file looks rather auto-generated, but I couldn't find any other references to where it comes from in the codebase. Do we just mutate the one which already exists?

Reply:
This is maintained manually and pushed to S3 as part of the release process: https://github.com/wagtail/wagtail/wiki/Creating-a-new-Wagtail-release#releaseswagtailio

(It used to be auto-generated with a command line tool as part of that process, but when we added the LTS / majorUrl bits it made more sense to edit it directly instead - https://github.com/wagtail/wagtail/commit/ec0daf604a366b18c842a13e1821391a5e2abdcd)

Reply:
Ah nice, thanks! Looking at the wiki:

> Edit scripts/latest.txt with the new version details

Looks like the existing process will catch the change so long as these changes are made, which is great!